### ISSUE TITLE
Couldn't able to login 

### ISSUE THREAD

**Issue by KalanaPerera** [instabot-py/instabot.py#1066]
> Fill in all requirements *before* submitting an issue.
> Do not delete any of the template, fill all of it in; even if you think it doesn't apply to your issue.
> If you fail to follow these simple instructions, we will close the ticket.

- [x] This is a checked box. **Do not leave spaces around the `x`!**

- [ x] I have searched for other issues with the same problem.

## Hosting
- [ x] Are you hosting the bot on a VPS or other Cloud hosting?
- [ ] If so; have you tried running the code on your local machine?

#### Select one:
- [ x] Bug
- [ ] Feature Request
- [ ] Technical Help
- [ ] Other

### Operating System? (include version)
 Linux 

## Python Version?
 3

## Description

Couldn't able to login  via Instagram bot

```
19.03.2018_09:33 Instabot v1.2.0 started at 19.03.2018 09:33:

19.03.2018_09:33 Trying to login as ******

19.03.2018_09:34 Login error! Check your login data!
19.03.2018_09:34 Removing already liked medias..
19.03.2018_09:34 Trying to unfollow #1: 
19.03.2018_09:34 Looks like there is nobody to unfollow.
19.03.2018_09:34 Removing already liked medias..
19.03.2018_09:34 Removing already liked medias..
19.03.2018_09:34 Removing already liked medias..
```

**Comment by jaybak** [comment]
I've started to get this also....

**Comment by mathiasadam** [comment]
Same here, any help?

**Comment by mbe9477** [comment]
same here... any idea?

**Comment by KalanaPerera** [comment]
yeah, I think because my IP Address is blocked by Instagram, Try to change IP and run, 

It's working for me..

**Comment by mbe9477** [comment]
You are right! I've used my iPhone instead of my local wifi and that worked fine again! 
After I've re-connected my Cable Router, I got a new IPv6 - but the bot still does not work!? 
That seems strange to me...

**Comment by KalanaPerera** [comment]
Mostly IP address are chaning 10 hrs time right? Better to use proxy will be good answer.
